enterprise/frontend/src/embedding-sdk-ee/tests/auth-flow/api-key.unit.spec.tsx

```diff
@@ -1,14 +1,19 @@
 import { waitForRequest } from "__support__/utils";
+import {
+  MOCK_API_KEY,
+  MOCK_INSTANCE_URL,
+} from "embedding-sdk-bundle/test/mocks/sso";
 import { defineMetabaseAuthConfig } from "embedding-sdk-package/lib/public/define-metabase-auth-config";
 
-import { MOCK_API_KEY, MOCK_INSTANCE_URL } from "../mocks/sso";
+import { setupEmbeddingSdkEnterprisePlugins } from "../support";
 
 import { setup } from "./setup";
 
-// Activates the plugins needed for the tests before any SDK code
-import "metabase-enterprise/sdk-plugins";
-
 describe("Auth Flow - API Key", () => {
+  beforeEach(() => {
+    setupEmbeddingSdkEnterprisePlugins();
```

Comment:
This function seems to only import `enterprise/frontend/src/metabase-enterprise/sdk-plugins.ts`, and it doesn't call `initializePlugins` defined there. Is this intended?

https://github.com/metabase/metabase/blob/2dbbb444fe6253727f0519b49d0c6ac9ef0b2acb/enterprise/frontend/src/metabase-enterprise/sdk-plugins.ts#L19-L25

Reply:
Thanks!
This is true, it's only activating the "sdk ee plugins", the plugins added to make this migration possible, I added a task to call that function call and a couple of lines of docs